CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
++++
+version = "1.4"
+aliases = ["/version/1/4"]
++++
```

Comment:
Does this metadata need to be present/is it shown in the Markdown preview?

Reply:
The instructions said to copy the file and replace the email address, so I did. You're right, though, it shows up in the Markdown rendering, which is ugly. `desktop/desktop` got rid of it, and so shall I.